README.md

```diff
@@ -1 +1,168 @@
 # macvtap CNI
+
+This plugin allows users to define Kubernetes networks on top of existing
+host interfaces. By using the macvtap plugin, the user is able to directly
+connect the pod interface to an host interface and consume it through a tap
+device.
+
+The main use case are virtualization workloads inside the pod driven by
+Kubevirt but it can also be used directly with QEMU/libvirt and it might be
+suitable combined with other virtualization backends.
+
+macvtap CNI includes a device plugin to properly expose the macvtap interfaces
+to the pods. A metaplugin such as [Multus](https://github.com/intel/multus-cni)
```

Comment:
not sure we want do delve into such detail, but, the DP is actually used to expose the tap devices. or is that implicit in the 'properly' ?

up to you, it reads well as is.

Reply:
Yes, I am actually trying not to go into much detail here further than introducing the presence of the device plugin. Using the term `macvtap interfaces` understood as a whole, single thing.